docs/specs/clients/core/identity/identity-keys.md

```diff
@@ -86,8 +61,8 @@ The CACAO is a standard JSON representation of a signed CAIP-122 message. For ex
   },
   "p": {
     "aud": "did:key:z6MkqJ6qV18zBazggzhGMHNgadEQGbX9RceEH3j2G6kNTbKq",
-    "iat": "2022-03-10T17:09:21.481+03:00",
-    "statement" : "I further authorize this DAPP to send and receive messages on my behalf for this domain using my WalletConnect identity.",
+    "iat": "2023-09-01T17:09:21.481+03:00",
+    "statement" : "I further authorize this app to send and receive messages on my behalf using my WalletConnect identity. Read more at walletconnect.com/identity",
```

Comment:
> Read more at walletconnect.com/identity
We need to publish this

Reply:
Alternatively we could point to our FAQ: https://walletconnect.com/faq. I'm fine with both tho

Reply:
Redirect added in https://github.com/WalletConnect/www-walletconnect/pull/104. Still need to write the final doc though, but this should be unblocked now.